### PR TITLE
add default gpu cost

### PR DESCRIPTION
**Pull request by srpomeroy**
## What does this PR change?
*  Adds the default price for a GPU so the ratio between CPU, RAM, and GPU can be properly calculated for Azure VMs.

## Does this PR relate to any other PRs?
* No

## How will this PR impact users?
* More accurate pricing estimates and allocation of resources for Kubernetes nodes that have GPUs in Azure

## Does this PR address any GitHub or Zendesk issues?
* No

## How was this PR tested?
* By using the API to set it.
```shell
curl --location --request POST 'localhost:9090/model/updateConfigByKey' \
    --header 'Content-Type: application/json' \
    --data-raw '{
  "GPU": "0.0428925"
}'
```

## Does this PR require changes to documentation?
* No

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
* Don't have access to do so

### CODE REVIEW

File: configs/azure.json [hunk 1]
```diff
@@ -2,17 +2,18 @@
     "provider": "Azure",
     "description": "Azure estimates based on April 2019 advertised prices",
     "CPU": "0.03900",
-    "spotCPU": "0.007764", 
-    "RAM": "0.001917", 
+    "spotCPU": "0.007764",
+    "RAM": "0.001917",
+    "GPU": "0.0428925",
```
Comment (mattray) [inline configs/azure.json:7]: What was the source of the price?

**Review by mattray — APPROVED** [review]

File: configs/azure.json (hunk 1 above)
Comment (srpomeroy) [reply configs/azure.json:7]: I did some math to figure out the price of a GPU for 3 different instance types in Azure then I averaged them. It seemed to make sense but probably best to ask an engineer to verify.

https://docs.google.com/spreadsheets/d/18Ugh-Di9zXp3mduOWpTAhiRx7kl-Lzh4wRQ3TFtCl1c/edit?usp=sharing

**Comment by mattray** [comment]
@srpomeroy if you can do the DCO I'll merge it

**Comment by srpomeroy** [comment]
> @srpomeroy if you can do the DCO I'll merge it

Stand by. I messed that up...

**Review by mattray — APPROVED** [review]

**Pull request merged**